### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by Luffytaro22**
### Implementing Desktop View, Mobile Menu and UI Interactions.
* Mobile and Desktop Home Page.
* Mobile and Desktop About Page.
* Mobile Menu.
* Links to both Pages in the header.
* Responsive Design.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,133 +1,167 @@
...
+					<div class="x"></div>
+				</div>
+				<ul id="menu">
+					<li class="menu-sections"><a href="index.html">Home</a></li>
+					<li class="menu-sections"><a href="About.html">About</a></li>
+					<li class="menu-sections">Program</li>
+					<li class="menu-sections">Join</li>
+					<li class="menu-sections">Sponsor</li>
+					<li class="menu-sections">News</li>
+					<li class="menu-sections">Campaign</li>
```
Comment (usorfaitheloho) [inline index.html:33]: Good job on implementing some of the project requirements, please as it is stated in the requirement, the `home` page should have a link to the `about` page and vice versa. You did well in implementing this feature for the `home `page but the `about` page doesn't seem to have a link to the `home` page. Kindly implement this feature so you can show how to make redirections to pages in your project 😄

File: index.html [hunk 2]
```diff
@@ -1,133 +1,167 @@
...
+					</div>
+					<div class="programs">
+						<img src="Images/program_icon_05.png" alt="landscape" />
+						<p class="title">Landscape</p>
+						<p>Appreciate the beautiful view of the capital with blooming cherry blossoms.</p>
+					</div>
+					<button type="button">Join Japanese Festival 2023</button>
+				</div>
+				<p id="hiden-program">SEE THE HOLE PROGRAM</p>
+			</section>
```
Comment (usorfaitheloho) [inline index.html:124]: You did well with this section implementation, please there is a `hover` effect for the programs selected, kindly implement it as given in the required design 😄 

Your design :
image

Design given :
image

File: Javascript/Activities.js [hunk 3]
```diff
@@ -3,39 +3,51 @@ const mainContainer = document.querySelector('#activities-container');
...
+    alt: 'architecture',
   },
   {
     activity: 'Hanami',
     type: '花見',
-    description: 'Hanami is the Japanese tradition of observing the beauty of flowers, but this word is generally associated with the period when the cherry blossoms are in bloom and when the Japanese flock to parks and gardens to gaze at their blossoms (sakura).',
+    description:
+      'Hanami is the Japanese tradition of observing the beauty of flowers<span class="text-hide">, but this word is generally associated with the period when the cherry blossoms are in bloom and when the Japanese flock to parks and gardens to gaze at their blossoms (sakura).</span>',
     img: 'Images/japanese-hanami.jpg',
+    alt: 'hanami',
```
Comment (usorfaitheloho) [inline Javascript/Activities.js:50]: Weldone in implementing the activities section using the dynamic javascript feature, Please in the design given there is a background image attached on each of them. Kindly implement this feature so looks close to the design given 😄 

Design given :
image

Your design :
image

File: About.html [hunk 4]
```diff
@@ -1,122 +1,166 @@
...
+			</section>
+			<section id="logo-section">
+				<div id="logo-explain">
+					<div class="title-div">
+						<h1>Japan Culture Festival 2023 Logo</h1>
+						<hr class="hr-bar">
+					</div>
+					<p
+						>The Logo of Japan Culture Festival 2023 was decided through the logo competition from
+						2.February to 3.March.<span class="text-hide2">After online public voting and Selection Committee's judging process, the logo from Naresh Agrawal won the prize.<br>After consultation with Mr. Agrawal we optimized the prize-winning-logo for this year's summit as following:</span></p
```
Comment (usorfaitheloho) [inline About.html]: You did well with the `about` page and it looks really close to the design, however, please implement these few changes listed below

1. The top section of this page should have a `background-image` as it is shown in the design which will give it a less simpler look compared to what we have with your design, Kindly implement this for a better UI 

Your design : 
image

Design given :
image

2. In the mobile view, the partners section background color is really darker compared to what we have in the given design. the design color used was `#414246` 

Your design :
image

Design given :
image

File: index.html [hunk 5]
```diff
@@ -1,133 +1,167 @@
...
+						<li class="nav-sections">Campaign</li>
+					</ul>
+				</nav>
+			</div>
+		</header>
+		<main>
+			<section id="title">
+				<div>
+					<h2>こんにちは世界</h2>
+					<h1>Japanese Culture Festival 2023</h1>
```
Comment (usorfaitheloho) [inline index.html]: Please a good HTML semantics rule is that `h1` tag should come before the `h2` tag in all your pages so as to help in Search engine optimization 👍

File: About.html [hunk 6]
```diff
@@ -1,122 +1,166 @@
...
+					</ul>
+				</nav>
+			</div>
+		</header>
+		<main>
+			<section id="title-about">
+				<div id="title-first">
+					<div class="title-div">
+						<h2>こんにちは世界</h2>
+						<h1>Japanese Culture Festival 2023</h1>
```
Comment (usorfaitheloho) [inline About.html]: Please a good HTML semantics rule is that h1 tag should come before the h2 tag in all your pages so as to help in Search engine optimization 👍

File: About.html [hunk 7]
```diff
@@ -1,122 +1,166 @@
...
+					<div class="partners">
+						<img src="Images/pngwing.com.png" alt="airbnb" class="logo" id="airbnb" />
+					</div>
+				</div>
+			</section>
+		</main>
+		<footer class="style-about">
+			<div id="logo-footer">
+				<img src="Images/torii (1).png" alt="logo" class="logo" />
+				<h1>Japan</h1>
```
Comment (usorfaitheloho) [inline About.html]: Please a good HTML semantics rule is that there should be only one h1 tag which should come before all other hx tags on all your pages so as to help in Search engine optimization 👍

File: index.html [hunk 8]
```diff
@@ -1,133 +1,167 @@
...
+			<section id="main-activities">
+				<div class="title-div">
+					<h2>Main Activities</h2>
+					<hr class="hr-bar">
+				</div>
+				<div id="activities-container"> </div>
+			</section>
+			<section id="partners" class="hide">
+				<div class="title-div">
+					<h1>Partner</h1>
```
Comment (usorfaitheloho) [inline index.html]: According to the design given in the mobile view, there is a `MORE`  button that selects the remaining activities listed, however this feature was not implemented in your design. Could you kindly add this feature to have a better UI 😄 

Your design :
image

Design given :
image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @Luffytaro22  ,
> 
> Great job for implemented the some of the Project Requirement , but there are some changes that need to be implemented in order to proceed to the next project 👍
> 
> ## Highlights 👏🏻
> ✔️ Home and About pages are close to the design given
    ✔️ Followed Javascript best practices.
    ✔️ Good Commit Messages
> 
> almost 

> ## Required Changes ♻️
> _Check the comments below the review._
> 

> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Luffytaro22) [reply index.html:33]: Excuse me, I was following these requirements:
image
So the link to the `home` page is in the logo header. Should I add a new `home menu` in the header next to the `About` one?

Comment (usorfaitheloho) [reply index.html:33]: In your About page ,in the navigation where you have the "About", it should be "Home" and it should link you to the homepage

**Review by Zuhaib042 — APPROVED** [review]
Hi @Luffytaro22 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Implemented all the suggested changes
✅ Beautiful design
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**